src/antsibull_changelog/cli.py

```diff
@@ -553,6 +568,23 @@ def lint_fragments(config: ChangelogConfig, fragments: List[ChangelogFragment],
     return 3 if messages else 0
 
 
+def command_lint_changelog_yaml(args: Any) -> int:
+    """
+    Lint changelogs/changelog.yaml file.
+
+    :arg args: Parsed arguments
+    """
+    errors = lint_changelog_yaml(
+        args.changelog_yaml_path, no_semantic_versioning=args.no_semantic_versioning)
+
+    messages = sorted(set(f'{error[0]}:{error[1]}:{error[2]}: {error[3]}' for error in errors))
+
+    for message in messages:
+        print(message)
+
+    return 3 if messages else 0
```

Comment:
Where are the return codes defined? Might be nice to have constants for readability (not for this PR, just a general suggestion).

Reply:
There is no definition. The only rule right now is that 0 means success, and anything else error.

Reply:
I've created #77 for this.